app/index.tsx

```diff
@@ -6,32 +6,27 @@ import { useFonts } from "expo-font";
 import config from "../tamagui.config";
 
 export default function App() {
-	const colorScheme = useColorScheme();
+  const colorScheme = useColorScheme();
```

Comment:
Please do not reformat files in this PR, open another PR if you want to suggest another formatting rules.

Reply:
Sorry about that, looks like it auto-formatted on save due to a lack of a prettier config setup.

Are you using tabs?

Reply:
Resolved